### PR TITLE
Update test suite for the new config

### PR DESCRIPTION
**Pull request by islamTaha12**

### CODE REVIEW

File: test/test_suite/test_cases/testcases_base.py [hunk 1]
```diff
@@ -55,11 +58,13 @@ def create_files(self, number_of_files):
...
         for i in range(0, number_of_files):
-            file_size = random.randint(1024, 1024 * 1024)
+            file_size = random.randint(1024, 1024*1024)
             file_path = '/tmp/upload/upload_file_%d' % random.randint(1, 1000)
 
-            with open(file_path, 'w') as file:
-                file.write(str(random.randint(0, 9)) * file_size)
+            with open(file_path, 'wb') as file:
+                for _ in range(0, file_size):
+                    file.write(os.urandom(1))
```
Comment (zaibon) [inline test/test_suite/test_cases/testcases_base.py]: This looks extremely inefficient. Why are you writing only 1 byte?

File: test/test_suite/config.ini [hunk 2]
```diff
@@ -1,3 +1,4 @@
 [main]
 number_of_servers = 4
 number_of_files = 5
+default_config_path = /gopath/src/github.com/zero-os/0-stor/client/cmd/zerostorcli/config.yaml
```
Comment (zaibon) [inline test/test_suite/config.ini]: Can you update the documentation to explain this line should be configued

Comment (islamTaha12) [reply test/test_suite/config.ini]: Done and I added a param in travis for it too

File: test/prepare_testing_env/docker_script.sh [hunk 3]
```diff
@@ -30,11 +30,13 @@ install_etcd(){
 }
 
 install_zerostor(){
-    go get -d github.com/zero-os/0-stor/server
-    cd /gopath/src/github.com/zero-os/0-stor/
-    git checkout ${ZERO_STORE_BRANCH}
-    cd server/
-    go build
+    go get -d github.com/zero-os/0-stor/cmd/zerostorserver
```
Comment (zaibon) [inline test/prepare_testing_env/docker_script.sh]: This block can be replace by just cd in the directory and call `make install`
`make install` will  build both the zerver and the cli and copy the binary in $GOPATH/bin

File: test/prepare_testing_env/run_tests.sh [hunk 4]
```diff
@@ -18,13 +18,13 @@ install_client(){
     echo "export PATH=/usr/local/go/bin:$PATH" >> ~/.bash_profile
 
     source ~/.bash_profile
-    go get -d github.com/zero-os/0-stor/client/cmd/zerostorcli
-    cd /gopath/src/github.com/zero-os/0-stor/client/cmd/zerostorcli
+    go get -d github.com/zero-os/0-stor/cmd/zerostorcli
```
Comment (zaibon) [inline test/prepare_testing_env/run_tests.sh]: Same remark with the Makefile

File: test/prepare_testing_env/docker_script.sh (hunk 3 above)
Comment (islamTaha12) [reply test/prepare_testing_env/docker_script.sh]: This script is running on docker container in packet machine, not in Travis one so we need to clone the repo at first.

File: test/prepare_testing_env/run_tests.sh (hunk 4 above)
Comment (islamTaha12) [reply test/prepare_testing_env/run_tests.sh]: done

**Pull request merged**